components/schema-migrator/migrations/director/202012081537_ord-service-initial.up.sql

```diff
@@ -0,0 +1,302 @@
+           jsonb_array_elements_text(expand.value) AS elements) AS package_labels
+UNION ALL
+(SELECT NULL::uuid         AS package_id,
+        api_definitions.id AS api_definition_id,
+        NULL::uuid         AS event_definition_id,
+        expand.key         AS key,
+        elements.value     AS value
+ FROM api_definitions,
+      jsonb_each(api_definitions.labels) AS expand,
+      jsonb_array_elements_text(expand.value) AS elements)
+UNION ALL
+(SELECT NULL::uuid     AS package_id,
+        NULL::uuid     AS api_definition_id,
+        id             AS event_definition_id,
+        expand.key     AS key,
+        elements.value AS value
+ FROM event_api_definitions,
+      jsonb_each(event_api_definitions.labels) AS expand,
+      jsonb_array_elements_text(expand.value) AS elements);
+
+CREATE VIEW tags AS
+SELECT *
+FROM (SELECT packages.id    AS package_id,
+             NULL::uuid     AS api_definition_id,
+             NULL::uuid     AS event_definition_id,
+             elements.value AS value
+      FROM packages,
+           jsonb_array_elements_text(packages.tags) AS elements) AS package_tags
+UNION ALL
+(SELECT NULL::uuid         AS package_id,
+        api_definitions.id AS api_definition_id,
+        NULL::uuid         AS event_definition_id,
+        elements.value     AS value
+ FROM api_definitions,
+      jsonb_array_elements_text(api_definitions.tags) AS elements)
+UNION ALL
+(SELECT NULL::uuid     AS package_id,
+        NULL::uuid     AS api_definition_id,
+        id             AS event_definition_id,
+        elements.value AS value
+ FROM event_api_definitions,
+      jsonb_array_elements_text(event_api_definitions.tags) AS elements);
+
+CREATE VIEW countries AS
+SELECT *
+FROM (SELECT packages.id    AS package_id,
+             NULL::uuid     AS api_definition_id,
+             NULL::uuid     AS event_definition_id,
+             elements.value AS value
+      FROM packages,
+           jsonb_array_elements_text(packages.countries) AS elements) AS package_countries
+UNION ALL
+(SELECT NULL::uuid         AS package_id,
+        api_definitions.id AS api_definition_id,
+        NULL::uuid         AS event_definition_id,
+        elements.value     AS value
+ FROM api_definitions,
+      jsonb_array_elements_text(api_definitions.countries) AS elements)
+UNION ALL
+(SELECT NULL::uuid     AS package_id,
+        NULL::uuid     AS api_definition_id,
+        id             AS event_definition_id,
+        elements.value AS value
+ FROM event_api_definitions,
+      jsonb_array_elements_text(event_api_definitions.countries) AS elements);
+
+CREATE VIEW package_links AS
+SELECT id                   AS package_id,
+       actions.type         AS type,
+       actions."customType" AS custom_type,
+       actions.url          AS url
+FROM packages,
+     jsonb_to_recordset(packages.package_links) AS actions(type TEXT, "customType" TEXT, url TEXT);
+
+CREATE VIEW api_resource_definitions AS
```

Comment:
Currently each API has only a single API Definition, once we implement the aggregator this view should be fully represented in a separate specifications table which will have the data as well